.github/workflows/test-m.yml

```diff
@@ -0,0 +1,70 @@
+name: Test
```

Comment:
Hey why is this file necessary?

Reply:
this was me just checking, the test specified in your repo, cause i wanted to make sure my implementation passes tests

Reply:
we shouldn't be merging this file in, can you please remove it?